stories/datepicker/datepicker.stories.js

```diff
@@ -1,1338 +1,2060 @@
-								<span class="fd-calendar__text">16</span>
-							</td>
-							<td class="fd-calendar__item" role="gridcell">
-								<span class="fd-calendar__text">17</span>
-							</td>
-							<td class="fd-calendar__item" role="gridcell">
-								<span class="fd-calendar__text">18</span>
-							</td>
-						</tr>
-						<tr class="fd-calendar__row">
-							<td class="fd-calendar__item" role="gridcell">
-								<span class="fd-calendar__text">19</span>
-							</td>
-							<td class="fd-calendar__item" role="gridcell">
-								<span class="fd-calendar__text">20</span>
-							</td>
-							<td class="fd-calendar__item" role="gridcell">
-								<span class="fd-calendar__text">21</span>
-							</td>
-							<td class="fd-calendar__item" role="gridcell">
-								<span class="fd-calendar__text">22</span>
-							</td>
-							<td class="fd-calendar__item" role="gridcell">
-								<span class="fd-calendar__text">23</span>
-							</td>
-							<td class="fd-calendar__item" role="gridcell">
-								<span class="fd-calendar__text">24</span>
-							</td>
-							<td class="fd-calendar__item" role="gridcell">
-								<span class="fd-calendar__text">25</span>
-							</td>
-						</tr>
-						<tr class="fd-calendar__row">
-							<td class="fd-calendar__item is-blocked" role="gridcell">
-								<span class="fd-calendar__text">26</span>
-							</td>
-							<td class="fd-calendar__item is-blocked" role="gridcell">
-								<span class="fd-calendar__text">27</span>
-							</td>
-							<td class="fd-calendar__item is-blocked" role="gridcell">
-								<span class="fd-calendar__text">28</span>
-							</td>
-							<td class="fd-calendar__item" role="gridcell">
-								<span class="fd-calendar__text">29</span>
-							</td>
-							<td class="fd-calendar__item" role="gridcell">
-								<span class="fd-calendar__text">30</span>
-							</td>
-							<td class="fd-calendar__item fd-calendar__item--other-month" role="gridcell">
-								<span class="fd-calendar__text">1</span>
-							</td>
-							<td class="fd-calendar__item fd-calendar__item--other-month" role="gridcell">
-								<span class="fd-calendar__text">2</span>
-							</td>
-						</tr>
-						</tbody>
-						</table>
-					</div>
-				</div>
-			</div>
-		</div>
-	</div>
+  <div class="fd-popover">
+    <div class="fd-popover__control">
+      <label class="fd-form-label" for="todayNavigationDateInput">en-US DatePicker with today navigation</label>
+      <div class="fd-input-group">
+        <input id="todayNavigationDateInput" class="fd-input fd-input-group__input" type="text" value="" placeholder="Pick a date">
+        <span class="fd-input-group__addon fd-input-group__addon--button">
+        <button class="fd-input-group__button fd-popover__control fd-button fd-button--transparent sap-icon--appointment-2"
+          aria-controls="Itcgq830"
+          aria-expanded="false"
+          aria-haspopup="true"
+          aria-label="Choose Date">
+        </button>
```

Comment:
Let's change it to:
```suggestion
          aria-label="Choose Date"></button>
```
Because new line is treated as a white space character

Reply:
THANK YOU. I spend unreal time to figure out what is wrong 😂